source/includes/administration/_tax_providers.md

```diff
@@ -0,0 +1,97 @@
+### Tax providers
+
+A tax provider enables a reseller to map tax codes to monetization products. The tax code provider type will ensure that invoices include taxes based 
+on the billing details (i.e. billing address) provided by the reseller. 
```

Comment:
What does` tax code provider type` means?
The tax code type or the tax code provider?

Reply:
`tax code provider` thats a typo. Fixed here [`eaac51d` (#425)](https://github.com/cloudops/cloudmc-api-docs/pull/425/commits/eaac51d7c8225b1aa3cd4c18922f5eb6f208ae70)